fury/ui.py

```diff
@@ -5240,3 +5240,165 @@ def _set_position(self, coords):
+        (_, image_height) = self._image_size
+        _text_box_size = (card_width - 2 * self.padding, (card_height *
+                          self.text_scale / 2) - (2 * self.padding))
+
+        self.image = ImageContainer2D(img_path=self.image_path,
+                                      size=self._image_size)
+
+        self.body_box = TextBlock2D(text=self.body,
+                                    size=_text_box_size,
+                                    color=self.text_color)
+
+        self.title_box = TextBlock2D(text=self.title, bold=True,
+                                     size=_text_box_size,
+                                     color=self.text_color)
+
+        _img_coords = (0, int(card_height - image_height))
+        _rem_space = int(_img_coords[1] / 3)
+        _title_coords = (self.padding, _rem_space - self.padding)
+        _text_coords = (self.padding, int(_title_coords[1] - _rem_space) -
+                        self.padding)
+
+        self.panel = Panel2D(size=self.card_size, color=(0.5, 0.5, 0.5))
+        self.panel.add_element(self.image, _img_coords)
+        self.panel.add_element(self.title_box, _title_coords)
+        self.panel.add_element(self.body_box, _text_coords)
+
+    def _get_actors(self):
+        """ Get the actors composing this UI component.
+        """
+
+        return self.panel.actors + self.image.actors \
+            + self.title_box.actors + self.body_box.actors
+
+    def _add_to_scene(self, _scene):
+        """ Add all subcomponents or VTK props that compose this UI component.
+        Parameters
+        ----------
+        scene : scene
+        """
+
+        self.panel.add_to_scene(_scene)
+        self.image.add_to_scene(_scene)
+        self.title_box.add_to_scene(_scene)
+        self.body_box.add_to_scene(_scene)
+
+    def _get_size(self):
+        return self.panel.size
+
+    def _set_position(self, _coords):
+        """ Position the lower-left corner of this UI component.
+        Parameters
+        ----------
+        coords: (float, float)
+            Absolute pixel coordinates (x, y).
+        """
+
+        self.panel.position = _coords
+
+    @property
+    def color(self):
+        """ Returns the background color of card.
+        """
+
+        return self.panel.color
+
+    @color.setter
+    def color(self, color):
+        """ Sets background color of card.
+        Parameters
+        ----------
+        color : list of 3 floats.
+        """
+
+        self.panel.color = color
+
+    @property
+    def body_text(self):
+        """ Returns the body text of the card.
+        """
+
+        return self.body_box.message
+
+    @body_text.setter
+    def body_text(self, text):
+        self.body_box.message = text
+
+    @property
+    def title_text(self):
+        """ Returns the title text of the card
+        """
+
+        return self.title_box.message
+
+    @title_text.setter
+    def title_text(self, text):
+        self.title_box.message = text
```

Comment:
Rename `body_text` to `body` and `title_text` to `title` to make the API look more pythonic.

You should also implement the `resize` method